src/elements/form/Control.ts

```diff
@@ -99,7 +99,11 @@ import { Helpers } from './../../utils/Helpers';
             <div class="error-message">
                 <span *ngIf="isDirty && errors?.required">{{control.label | uppercase}} is required</span>
                 <span *ngIf="isDirty && errors?.minlength">{{control.label | uppercase}} is required to be a minimum of {{ control.minlength }} characters</span>
-                <span *ngIf="isDirty && errors?.maxlength">{{control.label | uppercase}} is required to be a maximum of {{ control.maxlength }} characters</span>
+                <span class="character-limit-error" *ngIf="isDirty && maxLengthMet && focused && !errors?.maxlength">Sorry, you have reached the maximum character count of {{ control.maxlength }} for this field</span>
+                <span class="character-limit-error" *ngIf="errors?.maxlength">Sorry, you have exceeded the maximum character count of {{ control.maxlength }} for this field</span>
+                <span class="text-field-character-limit" [attr.data-automation-id]="text-area-character-count" *ngIf="focused && control.maxlength && (control.controlType=='text-area' || control.controlType=='textbox')">
```

Comment:
`[attr.data-automation-id]="text-area-character-count"` there is no property `text-area-character-count` on the class, did you want it to be just a string? if so then you will have to escape it`[attr.data-automation-id]="'text-area-character-count'"`